source/introduction.md

```diff
@@ -16,44 +21,57 @@ quadrupole, or a diffracting crystal. A lattice element may define a region in s
 distinguished by the presence of (possibly time-varying) electromagnetic fields,
 materials, apertures and other possible engineered structures. However, lattice elements
 are not restricted to being something physical and may, for example, just mark a particular point in space
-(EG: **Marker** elements), or may designate where beam lines intersect (**Fork** elements).
+(EG: `Marker` elements), or may designate where beam lines intersect (`Fork` elements).
 By convention, element names in PALS will be upper camel case.
 
-
+%---------------------------------------------------------------------------------------------------
+(s:branches)=
 ## Lattice Branches
 
-A lattice **branch** holds a collection of lattice elements. 
-There are two types of branches. One type, called a "tracking branch", holds
-an ordered array of lattice elements that gives a
-sequence of elements to be tracked through. A tracking branch can represent something like a
-storage ring, transfer line or Linac.
+A lattice `branch` holds a collection of lattice elements which is an ordered array of 
+lattice elements that gives the sequence of elements to be tracked through. 
+A branch can represent something like a storage ring, transfer line or Linac.
 In the simplist case, a program can track through the elements one element at a time.
 However, lattice elements may overlap which will naturally complicate tracking.
 
-The other type of branch, called a "lord" branch, is used to hold lattice elements that help describe:
-- Support elements (Girders)
-- Overlapping elements (Superposition)
-- Situations where an element is transversed multiple times as in an ERL or in opposite directions
-as in a colliding beam machine (Multipass)
-Lord branches will be explained in detail in later sections.
-
+%---------------------------------------------------------------------------------------------------
+(s:lattices)=
 ## Lattices
 
-A **lattice** is the root structure holding the information about a
+A `lattice` is the root structure holding the information about a
 ``machine``. A machine may be as simple as a line of elements (like the elements of a Linac) or
 as complicated as an entire accelerator complex with multiple storage rings, Linacs, transfer
 lines, etc.
 
-Essentially, a **lattice**, has an array of **branches** with each branch describing part of the
+Essentially a `lattice`, has an array of `branches` with each branch describing part of the
 machine. Branches can be interconnected to form a unified whole.
-Branches can be interconnected using **Fork** elements. 
+Branches can be interconnected using `Fork` elements. 
 This is used to simulate forking beam lines such as a connections to a transfer line, dump line, or an
-X-ray beam line. The **branch** from which other **branches** fork but is not forked to by any
-other branch is called a **root** branch.
+X-ray beam line.
```

Comment:
Here and elsewhere: replace "beam line" with "beamline"?

Reply:
Done.